src/js/exercisePopUpMarkup.js

```diff
@@ -0,0 +1,79 @@
+const exerciseDataPropertiesMapping = {
+  target: 'Target', bodyPart: 'Body Part', equipment: 'Equipment', popularity: 'Popular',
+};
+
+function createExerciseDataMarkup(exercise) {
+  let markup = '';
+
+  Object.keys(exerciseDataPropertiesMapping).forEach(key => {
+    if (exercise.hasOwnProperty(key)) {
+      markup += `
+        <li class="exercise-data-list-item">
+          <p class="exercise-data-parameter-name">${exerciseDataPropertiesMapping[key]}</p>
+          <p class="exercise-data-parameter-value">${exercise[key]}</p>
+        </li>
+      `;
+    }
+  });
+
+  if (exercise.hasOwnProperty('burnedCalories')) {
+    markup += `
+        <li class="exercise-data-list-item">
+          <p class="exercise-data-parameter-name">Burned Calories</p>
+          <p class="exercise-data-parameter-value">${exercise.burnedCalories}/${exercise.time} min</p>
+        </li>
+      `;
+  }
+
+  return markup;
+}
+
+const ratingStarSize = 18;
+
+function createRatingStarsMarkup(rating) {
+  const fullStar = `<svg class="exercise-rating-star" width="${ratingStarSize}" height="${ratingStarSize}" viewBox="0 0 32 32"><use href="../images/icons.svg#icon-star" style="fill: var(--yellow);"></use></svg>`;
```

Comment:
Does existing rating component not fit your needs?

Reply:
Will reuse component tomorrow